src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml

```diff
@@ -0,0 +1,267 @@
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    @if (!string.IsNullOrEmpty(Model.EventSummary))
+                    {
+                        @Html.MarkdownToHtml(@Model.EventSummary)
+                    }
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                        <a href="#" class="govuk-link govuk-link--no-visited-state @hideDetails">Change</a>
+                </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                    Does this event include guest speakers?
+                </dt>
+                <dd class="govuk-summary-list__value">
+                @{
+                    var hasGuestSpeakersText= @Model.HasGuestSpeakers.HasValue && @Model.HasGuestSpeakers.Value ? "Yes" : "No";
+                }
+                    @hasGuestSpeakersText
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                        <a href="#" class="govuk-link govuk-link--no-visited-state @hideDetails">Change</a>
+                </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                    Organiser name
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    @Model.OrganiserName
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                        <a href="#" class="govuk-link govuk-link--no-visited-state @hideDetails">Change</a>
+                </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                    Organiser email address
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    @Model.OrganiserEmail
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                        <a href="#" class="govuk-link govuk-link--no-visited-state @hideDetails">Change</a>
+                </dd>
+            </div>
+        </dl>
+        
+        
+        @if (Model.HasGuestSpeakers.HasValue && Model.HasGuestSpeakers.Value)
+        {
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        <h2 class="govuk-heading-m">
+                            Guest speakers
+                        </h2>
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                    <ol class="govuk-list govuk-list--number">
+                        @foreach (var speaker in Model.GuestSpeakers)
+                        {
+                            <li>
+                                @speaker.GuestName.Trim(), @speaker.GuestJobTitle.Trim()
```

Comment:
Are we not trimming where we are taking the input? If we ensure this then there is no need to use Trim anywhere else.

Reply:
I'll change it to the input

Reply:
@MarkFCain we need to Trim all the text inputs that we are capturing in creation journey.